composer.json

```diff
@@ -27,9 +28,12 @@
         "openeuropa/drupal-core-require-dev": "^8.7",
         "openeuropa/task-runner": "~1.0@beta",
         "phpunit/phpunit": "~6.0",
-        "symfony/dom-crawler": "~3.0",
+        "symfony/dom-crawler": "~3.4",
         "drupaltest/behat-traits": "dev-8.x-1.x"
     },
+    "_readme": [
+        "We use a minimum version of Pathauto 1.6 until a new release of oe_multilingual includes it. Then it can be removed."
+    ],
```

Comment:
Is this still needed? I mean, for lower deps we get pathauto 1.5 and drupal 8.7 and higher deps pathauto 1.6 and drupal 8.8. right?